apache.conf

```diff
@@ -0,0 +1,36 @@
+<VirtualHost *:443>
+    PassEnv SSP_APACHE_ALIAS
+    PassEnv APACHE_CERT_PATH
+    PassEnv APACHE_KEY_PATH
+    PassEnv HOST_NAME
+
+    #ServerAlias ${HOST_NAME}
```

Comment:
Is this used? If not, it can be removed to prevent confusion about its use in the future :)